modules/rest-api/pages/upload-retrieve-root-cert.adoc

```diff
@@ -21,15 +26,18 @@ GET /pools/default
 [#description]
 == Description
 
-The `POST` method and `/controller/uploadClusterCA` URI upload an appropriately configured certificate to the cluster, so that it becomes the root certificate for the cluster.
+The `POST` method and `/controller/uploadClusterCA` URI (which are deprecated in 7.1) upload an appropriately configured certificate to the cluster, so that it becomes the root certificate for the cluster.
 Either the Full Admin or the Security Admin role is required.
 The certificate must be specified as binary data, in a location on the filesystem, for successful upload to occur.
+In versions 7.1 and after, by default, this method and URI can be used on _localhost_ only.
+However, this default setting can be changed, by means of the `POST` method and `/settings/security/allowNonLocalCACertUpload` URI, with the `-d false` flag.
+The status of the current setting can be retrieved with the `GET` method and `/settings/security/allowNonLocalCACertUpload` URI.
 
-The `GET` method and `/pools/default/certificate` URI make a copy of the cluster's current root certificate, and returns the copy.
+The `GET` method and `/pools/default/certificate` URI (which are deprecated in 7.1) make a copy of the cluster's current root certificate, and return the copy.
```

Comment:
It's a bit tricky with this endpoint.
It's supposed to return one certificate, but in 7.1 it's possible to have several trusted certificates, so it's not clear what to return for backward compatibility.
Currently it works the following way in 7.1: after upgrade it continues to return the certificate that was used before the upgrade, but once at least one cert is uploaded it starts to return "400 Bad Request"  "deprecated, please use /pools/default/trustedCAs"

Reply:
I have modified.